### PR TITLE
Run tests with latest ruby version

### PR DESCRIPTION
**Pull request by ojiry**
👀

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,8 +18,8 @@ addons:
     - mysql-client-core-5.6
     - mysql-client-5.6
 rvm:
-  - 2.4.0-preview2
-  - 2.3.1
+  - 2.4.0-preview3
```
Comment (sodabrew) [inline .travis.yml]: Actually `2.4.0-rc1` is available starting today! I wonder if `2.4` works as an alias yet. I know `2.3` works as an alias for a floating most-recent ruby-2.3.x.

Comment (ojiry) [inline .travis.yml]: Oops! Thank you @sodabrew ! Updated `2.3`.
Let's try `2.4` alias 😋

**Comment by sodabrew** [comment]
Thank you!

**Comment by ojiry** [comment]
@sodabrew Thank you!

**Pull request merged**